data/tutorials/getting-started/3_01_ocaml_on_windows.md

```diff
@@ -6,8 +6,211 @@ description: >
 category: "Resources"
 ---
 
-There is a [DkML] Windows
-installer that we recommend for new users. However, while [DkML] has a modern OCaml 4.14.0 compiler,
+## opam
+
+Opam now features a fully native Windows compatible installation process that we
```

Comment:
I get the sentiment, but it isn't accurate to describe a Cygwin installation process as a fully native Windows compatible installation process. IMHO the real win is how easy it is to install Windows with opam 2.2 ... very few questions asked, no prereqs other than Git, etc. ... and the ease of install could be a good lead sentence.